sceptre/cli/helpers.py

```diff
@@ -93,6 +95,13 @@ def write(var, output_format="json", no_colour=True):
 
     click.echo(output)
 
+    if file_path:
+        dir_path = file_path.parent
```

Comment:
You're still treating a string like a pathlib.Path object here.

Reply:
Sorry, can you elaborate? I am not seeing the problem there.

Reply:
I've updated the type hints to clarify that a pathlib.Path is expected here.